components/RatingsTable/index.tsx

```diff
@@ -85,14 +87,53 @@ Props): JSX.Element {
     nextPage,
     previousPage,
     gotoPage,
+    allColumns,
+
+    setAllFilters,
+    setFilter,
     state: { pageIndex },
   } = table;
 
   const hasData = rows.length > 0;
 
+  // search logic
+  // no need for position for goalies
+  const searchTypes: Array<SearchType> = players[0] && 'blocker' in players[0] ? [
+    {text: 'Name', id: 'player-table-player'},
+    ] : [
+      {text: 'Name', id: 'player-table-player'},
+      {text: 'Position', id: 'player-table-position'},
+    ]
+
+  const [ searchType, setSearchType ] = useState(searchTypes[0].id);
+  const [ searchText, setSearchText ] = useState('');
+
+  const updateFilter = (text) => {
+    console.log(allColumns);
```

Comment:
this isnt necessary anymore

Reply:
you are correct.  I missed one.  updated